src/core/Akka.Persistence/AtLeastOnceDelivery.cs

```diff
@@ -159,7 +159,7 @@ protected override void OnReplaySuccess()
         /// <param name="receive">TBD</param>
         /// <param name="message">TBD</param>
         /// <returns>TBD</returns>
-        protected override bool AroundReceive(Receive receive, object message)
+        protected internal override bool AroundReceive(Receive receive, object message)
```

Comment:
It is not affect a public API. But we need it if we want to enable [InternalVisibleTo] for this assembly